add_to_pydotorg.py

```diff
@@ -333,6 +333,29 @@ def has_sigstore_signature(filename: str) -> bool:
             os.path.exists(filename + ".sig") and os.path.exists(filename + ".crt")
         )
 
+    # Ensure that Sigstore CLI installed on the download server is
+    # at least v3.0.0 or later to ensure valid Sigstore bundles are generated.
+    try:
+        sigstore_version_stdout = subprocess.check_output(
+            ["python3", "-m", "sigstore", "--version"]
+        )
+        sigstore_version_match = re.search(
+            r"([0-9][0-9.]*[0-9])", sigstore_version_stdout.decode()
+        )
+        if not sigstore_version_match:
+            error(
+                f"Couldn't determine version of Sigstore CLI: {sigstore_version_stdout.decode()}"
+            )
+        sigstore_version = sigstore_version_match.group(1)
+        sigstore_major_version = int(sigstore_version.partition(".")[0])
+        if sigstore_major_version < 3:
+            error(
+                f"Sigstore v3 or later must be installed (currently {sigstore_version}), run python -m pip install -r requirements.txt"
```

Comment:
Wrapping long line, and add colon after "run":
```suggestion
                f"Sigstore v3 or later must be installed "
                f"(currently {sigstore_version}), "
                f"run: python -m pip install -r requirements.txt"
```